azure-pipelines.yml

```diff
@@ -45,8 +45,6 @@ stages:
       parameters:
         MirrorRepo: templating
         LclSource: lclFilesfromPackage
-        MirrorBranch: $(OneLocBuildBranch)
-        LclPackageId: 'LCL-JUNO-PROD-TEMPLATING'
         ${{ if eq(variables['Build.SourceBranch'], format('{0}{1}', 'refs/heads/', variables['OneLocBuildBranch'])) }}:
           MirrorBranch: $(OneLocBuildBranch)
           LclPackageId: 'LCL-JUNO-PROD-TEMPLATING'
```

Comment:
With some tries, I didn't figure out why https://github.com/dotnet/templating/pull/5266 didn't get `LclPackageId: 'LCL-JUNO-PROD-TEMPLATING'` passed to OneLocBuild successfully.

- First, the condition should be satisfied for release/7.0.1xx.
The change https://github.com/dotnet/templating/pull/5266 started the task OneLocBuild in internal run. It said `condition: or(eq(variables['Build.SourceBranch'], format('{0}{1}', 'refs/heads/', variables['OneLocBuildBranch'] )), eq(variables['Build.SourceBranch'], 'refs/heads/main'))` is true. Actually `eq(variables['Build.SourceBranch'], format('{0}{1}', 'refs/heads/', variables['OneLocBuildBranch'] ))` should be true. So the conditional parameter should be passed.
- Second, OneLocBuild task should get passed values as main branch also uses conditional parameters but doesn't have this problem.

I'd like to output the parameters/variables in the real internal run for investigation since there is no other way to simulate.